From: Julian Taylor <jtaylor.debian@googlemail.com>
Date: Thu, 22 Jun 2017 07:35:29 +0200
Subject: increase tabheight

Increase tabheight further to workaround crash when changing database
tabs.
Bug-Ubuntu: https://bugs.launchpad.net/bugs/1659159
Bug: https://sourceforge.net/p/keepass/bugs/1634
---
 KeePass/Forms/MainForm.cs | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: keepass2/KeePass/Forms/MainForm.cs
===================================================================
--- keepass2.orig/KeePass/Forms/MainForm.cs
+++ keepass2/KeePass/Forms/MainForm.cs
@@ -118,7 +118,7 @@ namespace KeePass.Forms
 
 			if(!Program.DesignMode)
 			{
-				if(MonoWorkarounds.IsRequired(891029)) m_tabMain.Height += 5;
+				if(MonoWorkarounds.IsRequired(891029)) m_tabMain.Height += 10;
 
 				SuspendLayoutScope sls = new SuspendLayoutScope(false,
 					m_menuMain, m_ctxGroupList, m_ctxPwList, m_ctxTray);
